### ISSUE TITLE
Founding member

### ISSUE THREAD

**Issue by BarryNorton** [motools/musicontology#10]
Create a representation of founding members, perhaps either (Barry's initial suggestion) as a subproperty of mo:member or (Yves' refinement) as a subclass of mo:Membership.

**Comment by boferri** [comment]
I would rather suggest the second one.

**Comment by BarryNorton** [comment]
It raises an interesting question though.

To motivate Membership as an event, consider Robbie Williams. He was a founding member of Take That, then he left, then he came back (a separate event).

Now is his second membership a founding membership, or not? (Perhaps not if we call the event that: FoundingMembership)

If not, do we constrain (if we were to fully axiomatise) the earliest Membership event between two given Artists to be the only one allowed in the FoundingMembership subclass?

**Comment by boferri** [comment]
Well an artist could be part of one FoundingMembership event and several 
Membership events per group.
I wouldn't suggest (more) constraints as a group could also be founded 
by more than two members, or?

**Comment by BarryNorton** [comment]
The point of the would-be constraint is not that Robbie should preclude Gary's also having founded the band, but that Robbie's re-joining raises the possibility that one models his first membership without classifying that as a founding membership and his second one as such. Which is odd. 

In fact, the more I think about it, the more I believe if you were a founder you're a founder for life, whether you have a current membership or not. This argues against the Membership subclass, but also argues against the simple subproperty unless mo:member were clarified to mean 'has at some point been a member', rather than being something like:
http://dbpedia.org/ontology/currentMember

(That this feature is underspecified is consistent with its being marked unstable)

**Comment by boferri** [comment]
Yep. You are right in both points. So we should redefine mo:member to something like 'has at some point been a member' and maybe introduce more sub properties of mo:member such as mo:current_member and mo:founding_member. Finally, you can utilise the short cut descriptions (via mo:member and sub properties) and the more detailed ones (via mo:Membership) side by side.
